src/olympia/abuse/tests/test_cinder.py

```diff
@@ -1214,23 +1220,55 @@ def test_appeal_anonymous(self):
 
     def test_appeal_logged_in(self):
         addon = self._create_dummy_target()
-        self._test_appeal(CinderUser(user_factory()), self.CinderClass(addon))
+        self._test_appeal(
+            CinderUser(user_factory()), cinder_entity_instance=self.CinderClass(addon)
+        )
         assert (
             addon.current_version.needshumanreview_set.get().reason
             == NeedsHumanReview.REASONS.ADDON_REVIEW_APPEAL
         )
         assert addon.current_version.reload().due_date
 
-    def test_appeal_specific_version(self):
+    def test_appeal_specific_version_from_report(self):
```

Comment:
Can you add docstrings for these tests?

Regarding `test_appeal_specific_version_from_report` you mentioned a report.. I think that maps to the CinderClass instance you've defined that (specified a version) right? And therefore we expect the original version (current_version) not to be flagged, but only the one referenced in the CinderClass.

Regarding `test_appeal_specific_version_from_action` I'm a bit lost on what exactly this tests...

Reply:
I've attempted to do this in https://github.com/mozilla/addons-server/pull/22975/commits/8486ff498480e1df40103df854be42ef02a1eb6a - though the docstrings are pretty much what the comments at the top of the tests already said... let me know if this helps...